source/base/parameter_handler.cc

```diff
@@ -2379,6 +2396,44 @@ ParameterHandler::operator==(const ParameterHandler &prm2) const
 }
 
 
+std::set<std::string>
+ParameterHandler::get_entries_wrongly_not_set() const
+{
+  std::set<std::string> entries_wrongly_not_set;
+
+  for (auto it : entries_set_status)
+    if (it.second.first == true && it.second.second == false)
+      entries_wrongly_not_set.insert(it.first);
+
+  return entries_wrongly_not_set;
+}
+
+void
+ParameterHandler::assert_that_entries_have_been_set() const
+{
+  std::set<std::string> entries_wrongly_not_set =
+    this->get_entries_wrongly_not_set();
+
+  if (entries_wrongly_not_set.size() > 0)
+    {
+      std::string list_of_missing_parameters = "\n\n";
+      for (auto it : entries_wrongly_not_set)
+        list_of_missing_parameters += it + "\n";
```

Comment:
Let's put a couple of spaces in front to offset them from the surrounding text.
```suggestion
        list_of_missing_parameters += "   " + it + "\n";
```